src/components/FlowRunWorkQueue.vue

```diff
@@ -1,16 +1,24 @@
 <template>
   <div class="flow-run-work-queue">
-    <span>Work Queue</span> <WorkQueueIconText :work-queue-name="workQueueName" :work-pool-name="workPoolName" />
+    <span>Work Queue</span>
+    <WorkQueueIconText :work-queue-name="workQueueName" :work-pool-name="workPoolName" />
+    <WorkQueueStatusIcon v-if="isNotTerminal && workPoolName" :work-queue-name="workQueueName" :work-pool-name="workPoolName" />
   </div>
 </template>
 
 <script lang="ts" setup>
+  import { computed } from 'vue'
   import WorkQueueIconText from '@/components/WorkQueueIconText.vue'
+  import WorkQueueStatusIcon from '@/components/WorkQueueStatusIcon.vue'
+  import { isTerminalStateType } from '@/models'
 
-  defineProps<{
+  const props = defineProps<{
     workQueueName: string,
     workPoolName?: string | null,
+    flowRunState?: string | null,
   }>()
+
+  const isNotTerminal = computed(() => props.flowRunState && !isTerminalStateType(props.flowRunState))
```

Comment:
I've added a check for state here because the health of the work queue is only really relevant on upcoming and in progress runs.  Showing an unhealthy work queue icon for a run that has successfully completed feels confusing to me.